Jenkinsfile

```diff
@@ -183,15 +183,20 @@ try {
         stage('Parallel test run') {
             set_gh_status_pending(is_pr, 'Testing')
             timestamps {
-                parallel test_runs
+                try {
+                    parallel test_runs
+                } catch (Exception e) {
+                    currentBuild.result = 'UNSTABLE'
+                }
             }
         }
     } // if testinfo_sumz
 
 } catch (Exception e) {
     echo "Error: ${e}"
-    // Set currentBuild.result as FAILURE if there is an error at any point
-    currentBuild.result = 'FAILURE'
+    if (currentBuild.result != 'UNSTABLE')
+        // Set currentBuild.result as FAILURE if there is an error in building
```

Comment:
It can be 'null' so that's why it's set to FAILURE. And if it's UNSTABLE we wan't to keep it that way because that tells us that there wass an error in testing setup and not in the PR/master build.